File: src/main/java/org/customer/dto/TransactionDTO.java
package org.customer.dto;

import lombok.Builder;
import lombok.Data;
import lombok.Getter;

import java.util.Date;


@Data
@Getter
@Builder
public class TransactionDTO {
    private String id;
    private String accountId;
    private String customerId;
    private double amount;
    private Date timestamp;
}

File: src/main/java.kotlin/org/customer/dto/TransactionDTO.kt
package org.customer.dto

import lombok.Builder
import lombok.Data
import lombok.Getter
import java.util.*

@Data
@Getter
@Builder
class TransactionDTO {
    private val id: String? = null
    private val accountId: String? = null
    private val customerId: String? = null
    private val amount = 0.0
    private val timestamp: Date? = null
}
